### PR TITLE
update release template for 7.11

### PR DESCRIPTION
**Pull request by nickboldt**
update release template for 7.11 - https://github.com/eclipse/che/issues/16478

Change-Id: Id1e37e40931178b4c6ade68261a5cf7677a13b1f
Signed-off-by: nickboldt <nboldt@redhat.com>

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -15,28 +15,28 @@ RELEASE-TEMPLATE-BEGIN
...
-  - [ ] _specific version of theia used: *(tag or sha)*_
 - [ ] che-machine-exec
-- [ ] che-plugin-registry _(depends on che-theia, che-machine-exec)_
+    - [ ] che-plugin-registry _(depends on che-theia, che-machine-exec)_
 - [ ] che-devfile-registry
 - [ ] che-parent
-- [ ] che-docs _(depends on parent)_
-- [ ] che _(depends on docs and parent)_
-- [ ] che-operator _(depends on all of the above)_
+    - [ ] che-docs _(depends on parent)_
```
Comment (mkuznyetsov) [inline RELEASE.md:34]: new items "che-dashboard" and "che-workspace-loader" should be added alongside "che-docs" (depending on parent), as they're now need before the next step

Comment (nickboldt) [reply RELEASE.md:34]: OK, please edit my PR then.

Comment (mkuznyetsov) [inline RELEASE.md:34]:
```suggestion
    - [ ] che-docs _(depends on parent)_
    - [ ] che-dashboard _(depends on parent)_
    - [ ] che-workspace-loader _(depends on parent)_
```

File: RELEASE.md [hunk 2]
```diff
@@ -15,28 +15,30 @@ RELEASE-TEMPLATE-BEGIN
...
+    - [ ] che-dashboard _(depends on parent)_
+    - [ ] che-workspace-loader _(depends on parent)_
+        - [ ] che + quay images _(depends on docs and parent)_
```
Comment (mkuznyetsov) [inline RELEASE.md]:
```suggestion
        - [ ] che + quay images _(depends on docs, dashboard, workspace-loader and parent)_
```

**Comment by nickboldt** [comment]
Looks good, thanks for updates. Now we just need approval :)

File: RELEASE.md [hunk 3]
```diff
@@ -15,28 +15,30 @@ RELEASE-TEMPLATE-BEGIN
...
 In parallel, the following releases can be done:
 
-* che-theia,
-* che-machine-exec,
-* che-devfile registry, then che-plugin-registry (once che-theia and machine-exec are done)
-* che-parent, then che-docs, then che as Release Candidate for QE
+* che-theia
+* che-machine-exec
+    * che-plugin-registry (once che-theia and machine-exec are done)
+* che-devfile registry
```
Comment (amisevsk) [inline RELEASE.md:21]:
```suggestion
* che-devfile-registry
```

File: RELEASE.md [hunk 4]
```diff
@@ -15,28 +15,30 @@ RELEASE-TEMPLATE-BEGIN
...
+* che-parent, then che-docs, then che (including [quay images](https://github.com/eclipse/che/blob/master/.ci/cico_common.sh#L123-L131)) to release-candidate branch
 
 Then in series:
 
-* che-parent, then che-docs, then che as Release
+* che-parent, then che-docs, then che (copy from release-candidate to release branch)
```
Comment (amisevsk) [inline RELEASE.md:26]: ~~I'm not sure I understand the difference between `che-parent, then che-docs, then che` above, followed by the same but in series.~~

Comment (amisevsk) [reply RELEASE.md:26]: Ah nevermind, misread line 22.

**Review by amisevsk — APPROVED** [review]

**Review by vparfonov — APPROVED** [review]

**Pull request merged**